### DISCUSSION TITLE
Feature to Save Filters

### DISCUSSION THREAD

**Discussion by jeffcwolf** [dbgate/dbgate#867] [Ideas]
I would love the ability to save a set of filters so that I can automatically choose a saved filter to apply to a table, making the ability to edit a specific subset of table data easy to access. This can't be done with Views or Perspectives, as far as I can tell, because these are both read-only. One could write SQL Select queries to do this, but a Saved Filter(s) feature would make this faster. If I am missing something - and this can already be done (I.e. a way to save or open an editable data view), I would love to know. Thanks for the great application.

**Comment by janproch** [comment]
Good idea, I will think about it. Technically it is not problem, but I am not sure, where to place this functionality, so that UX won't become too much complicated. Any ideas?
